ui/templates/base.html

```diff
@@ -15,6 +15,15 @@
     <script type="text/javascript">
     var SETTINGS = {{ js_settings_json|safe }};
     var CURRENT_PAGE_URL = '{{ request.build_absolute_uri }}';
+    {% if SETTINGS.gtmID %}
+      <!-- Google Tag Manager -->
+      <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Need to remove this script tag and the trailing one since it's already in a `<script/>` block